docs/Installation.md

```diff
@@ -0,0 +1,101 @@
+# Installation Guide for ClusterLink
+The ClusterLink gateway contains three main components: the control-plane, data-plane, and gwctl (more details can be found [here](../README.md#what-is-clusterlink)).
+
+ClusterLink can be deployed in a local environment (e.g., Kind) or any remote K8s cluster environment (e.g., google GKE, amazon EKS, IBM IKS, etc.).
+
+To deploy the ClusterLink gateway in a K8s cluster, follow these steps:
+
+1. Create and deploy certificates and the ClusterLink deployment YAML file.
+2. Expose the ClusterLink deployment to a public IP.
+3. Optionally, create a central gwctl component to control one or more gateways.
+
+Before you begin, please build the project according to the [instructions](../README.md#building-clustelink).
+### 1. ClusterLink deployment and certificates
+In this step, we create the ClusterLink certificates and deployment files.
+
+1) Create a folder (eg. DEPLOY_DIR) for all deployments and yaml files:
+
+        export PROJECT_DIR=`git rev-parse --show-toplevel`
+        export DEPLOY_DIR=~/clusterlink-deployment
+        mkdir -p $DEPLOY_DIR
+        cd $DEPLOY_DIR
+
+2) Create Fabric certificates, including the Root CA certificate for all the clusters (This step needs to be done only once per fabric).:
+
+        $PROJECT_DIR/bin/cl-adm create fabric
+
+    Note: This step needs to be done only once per fabric.
+3) Create certificates and a deployment file for the cluster (e.g., peer1):
+
+        $PROJECT_DIR/bin/cl-adm create peer --name peer1
+
+    Note: By default, the images are pulled from ghcr.io/clusterlink-net. To change the container registry, you can use the ```--container-registry``` flag.  
```

Comment:
qq: is ghcr.io where we upload images to? I was under the impression that we wanted to use quay.io?

Reply:
Yes, but in the last meeting, @zivnevo propose to use ghcr so we add automation (upload new image per release).